flagd/cmd/start.go

```diff
@@ -63,6 +67,9 @@ func init() {
 	flags.StringP(otelCollectorURI, "o", "", "Set the grpc URI of the OpenTelemetry collector "+
 		"for flagd runtime. If unset, the collector setup will be ignored and traces will not be exported.")
 
+	flags.BoolP(syncEnabledFlagName, "e", false, "Set true to enable gRPC sync service from flagd. "+
```

Comment:
```suggestion
	flags.BoolP(syncEnabledFlagName, "e", false, "Enables the gRPC sync service. "+
```

Reply:
I could go either way on this one. I was trying to make it sound similar to the other description.

Reply:
I like the suggestion, fixed with commit https://github.com/open-feature/flagd/pull/1237/commits/bf818c7a163d4b019e408fce9653bcc4a72aa1b4 as we need to update documentation as well